From 63a11e5f1f4d700230293cc736c6532ff032a49d Mon Sep 17 00:00:00 2001
From: Gowtham Suresh Kumar <gowtham.sureshkumar@arm.com>
Date: Thu, 2 Mar 2023 09:24:14 +0000
Subject: [PATCH] Update tss-esapi version

The tmp2-tss version on meta-security has been updated to 4.0.1
from 3.2.0. This is not compatible with the current tss-esapi crate.
The newer version 7.2.0 of the crate is now available which
resolves the issue.

Signed-off-by: Gowtham Suresh Kumar <gowtham.sureshkumar@arm.com>

GuillaumeG: Drop Cargo.lock part since osc service already update it
---
 Cargo.lock           | 8 ++++----
 Cargo.toml           | 2 +-
 e2e_tests/Cargo.toml | 2 +-
 3 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/Cargo.toml b/Cargo.toml
index b6327dc8..87cc1a10 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -29,7 +29,7 @@ log = { version = "0.4.14", features = ["serde"] }
 cryptoki = { version = "0.3.0", optional = true, features = ["psa-crypto-conversions"] }
 picky-asn1-der = { version = "0.2.4", optional = true }
 picky-asn1 = { version = "0.3.0", optional = true }
-tss-esapi = { version = "7.1.0", optional = true }
+tss-esapi = { version = "7.2.0", optional = true }
 bincode = "1.3.1"
 structopt = "0.3.21"
 derivative = "2.2.0"
diff --git a/e2e_tests/Cargo.toml b/e2e_tests/Cargo.toml
index 3302ce38..b895e84b 100644
--- a/e2e_tests/Cargo.toml
+++ b/e2e_tests/Cargo.toml
@@ -17,7 +17,7 @@ log = "0.4.14"
 rand = "0.7.3"
 env_logger = "0.8.3"
 stdext = "0.3.1"
-tss-esapi = { version = "7.1.0", optional = true }
+tss-esapi = { version = "7.2.0", optional = true }
 
 [dev-dependencies]
 ring = "0.16.20"
